### PR TITLE
Only Store sucessful builds

### PR DESCRIPTION
**Pull request by dabates**
Adds the Status to the return json, the model and the database.

This might be a little overkill, but it will make it easier to see if somehow a broken artifact got in.

We change the linq expression to only pull success Status builds.

### CODE REVIEW

File: LidarrAPI/Database/Migrations/20180201022530_AddStatusField.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LidarrAPI.Database.Migrations
+{
+    public partial class AddStatusField : Migration
```
Comment (danielunderwood) [inline LidarrAPI/Database/Migrations/20180201022530_AddStatusField.cs:8]: Should this have some type of identifier for ordering migrations? Or is that handled by the migration attribute of the `Designer.cs` file?

**Review by danielunderwood — COMMENTED** [review]
Most of it looks good from what I can tell. See the comment on migrations, but it may be the way it's supposed to be.

File: LidarrAPI/Database/Migrations/20180201022530_AddStatusField.cs (hunk 1 above)
Comment (dabates) [reply LidarrAPI/Database/Migrations/20180201022530_AddStatusField.cs:8]: This should be tracked by the designer and there is a migration table in the DB .. that shows order of migrations :)

**Review by danielunderwood — APPROVED** [review]
Approved on the code side. Since we aren't doing a ton of changes on this repo, I've swapped the version we're hosting to this branch to make sure nothing funny comes up in the next day or so.

**Pull request merged**